### PR TITLE
update readme, comments & add isDts

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Update the readme with a simple getting started
Add comments on public api
Add `isDts` on symbols to easily check if the symbol comes from node_modules or not.

### CODE REVIEW

**Comment by GrandSchtroumpf** [comment]
I also updated the package.json : 
- change github url to ng-ast
- add myself as contributor (I hope that's ok)

File: README.md [hunk 1]
```diff
@@ -4,13 +4,44 @@
...
+import { join } from 'path';
+import { WorkspaceSymbols } from 'ngast';
+
+const config = join(process.cwd(), 'tsconfig.json');
+const workspace = new WorkspaceSymbols(config);
+```
+
+Then you can analyse your project. For that run `analyzeAll`: 
+```typescript
+workspace.analyzeAll();
```
Comment (irustm) [inline README.md:26]:
```ts
 error TS2341: Property 'analyzeAll' is private and only accessible within class 'WorkspaceSymbols'.

21     workspace.analyzeAll();

```

**Comment by GrandSchtroumpf** [comment]
@irustm thanks, I remove this part from the readme.

File: README.md [hunk 2]
```diff
@@ -4,13 +4,39 @@
...
+const modules = workspace.getAllModules();
+const components = workspace.getAllComponents();
+const directives = workspace.getAllDirectives();
+const injectables = workspace.getAllInjectable();
+const pipes = workspace.getAllPipes();
+```
+
+The **first time** one of the method above is called, `ngast` will run the analysis of the workspace.
+
+> The analysis is currently quite long: >10sec for a small project can go beyond 2min for a very large project.
```
Comment (mgechev) [inline README.md:35]: Maybe bold instead of use a quote?

**Review by mgechev — APPROVED** [review]

**Comment by GrandSchtroumpf** [comment]
@mgechev I've updated after your comment. I don't have write access on this repo so I cannot merge. Could you merge it ?

**Pull request merged**